pkg/model/crd_test.go

```diff
@@ -61,34 +61,53 @@ func TestSNSTopic(t *testing.T) {
 	specFields := crd.SpecFields
 	statusFields := crd.StatusFields
 
+	// The SNS Topic uses an "Attributes" map[string]*string to masquerade
```

Comment:
Dumb question:
Why do we need to break Map attributes into String Fields?
Why not just keep them as a Map in ko.Spec?